packages/twenty-front/src/modules/object-record/field/types/FieldMetadata.ts

```diff
@@ -118,7 +118,16 @@ export type FieldCurrencyValue = {
   amountMicros: number | null;
 };
 export type FieldFullNameValue = { firstName: string; lastName: string };
-export type FieldRatingValue = '1' | '2' | '3' | '4' | '5';
+
+export enum FieldRatingValue {
+  ZERO = 'ZERO',
+  ONE = 'ONE',
+  TWO = 'TWO',
+  THREE = 'THREE',
+  FOUR = 'FOUR',
+  FIVE = 'FIVE',
+}
+
```

Comment:
When creating enums on the backend, we are using the value string as name itself. 

https://github.com/twentyhq/twenty/blob/96264e264ccfaaaa1a7290e1b3fab91deb5f4f40/packages/twenty-server/src/workspace/workspace-schema-builder/factories/enum-type-definition.factory.ts#L75-L83

```
"name": {
  value: "name",
  description: ......
}
```
So when I pass numbers as string, like - 
```
"0": {
  value: "0",
  description: ......
}
```
 GraphQLEnumType has an internal validation where `Names must match /^[_a-zA-Z]`. Hence I had to pass it as a long form string numbers.

Reply:
@AdityaPimpalkar Good catch ! 
Actually `name` and `value` can differ, so maybe we can put a number inside `value` and see how it's handled by GraphQL

Reply:
@magrinj You're correct! we can pass number inside `value` but I think you misunderstood problem. The enum `name` is being assigned SAME as the `value` coming from the front end. 

Let's say if we create an option from the front end named `Option 1`. The enum created looks like this-
```
{
  OPTION_1: { value: 'OPTION_1', description: 'Option 1' },
}
```
In rating values case (we are passing hardcoded options), let's say we pass an option as `0`. So it will look something like this-
```
{
  0: { value: '0', description: '0' },
}
```
Since the Name of enum needs to have letters or underscore, this throws error
Hope this wasn't confusing 😃